data/varad-kulkarni172.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "Varad Kulkarni",
+  "bio": "Inquisite Full Stack Web Developer who aims to make the paperless education more reliable and easy to use for people...I am also interested in Artificial Intelligence and data analytics."
```

[Recommendation/Question] What do you mean by “paperless education”? Do you mean online tech courses? 🤔 
```suggestion
  "bio": "Inquisitive Full Stack Web Developer who aims to make the paperless education more reliable and easy to use for people. I am also interested in Artificial Intelligence and Data Analytics."
```

---

promoting paperless education by creating applications

---

> promoting paperless education by creating applications

Oh ok that makes sense :)